frontend/packages/topology/locales/en/topology.json

```diff
@@ -1,8 +1,8 @@
 {
-  "Move Connector": "Move Connector",
-  "Delete Connector": "Delete Connector",
-  "Delete Application": "Delete Application",
-  "Edit Application Grouping": "Edit Application Grouping",
+  "Move connector": "Move connector",
+  "Delete connector": "Delete connector",
+  "Delete application": "Delete application",
+  "Edit application grouping": "Edit application grouping",
   "Edit {{applicationName}}": "Edit {{applicationName}}",
   "Select an Application": "Select an Application",
   "Create Application": "Create Application",
```

Comment:
Some sections in this file have `application`.

Reply:
yeah for Kubernetes resources we should use title case so there is confusion for Application and Project if used in direct context. Even @andrewballantyne highlighted it LMK what you guys think have title case in direct context i.e create, delete , select etc or follow sentence case

Reply:
For this case I would use title case for Project and Application since they are considered proper terms in the [OpenShift glossary](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc).